crates/ruma-common/src/events/file.rs

```diff
@@ -24,7 +27,7 @@ pub struct FileEventContent {
     pub message: MessageContent,
 
     /// The file content of the message.
-    #[serde(rename = "org.matrix.msc1767.file")]
+    #[serde(rename = "m.file")]
```

Comment:
This change seems unrelated, why have (only) the stable string here?

Reply:
I was hoping to reuse the new events for the transitional events but that didn't work out.

These event will only be used after the transitional events so they will never have the unstable prefix. This can probably be done in a separate commit/PR if you prefer.

Reply:
Wouldn't people want to experiment with the full thing after the transition now, and use the unstable prefixes for that? This seems like a case where we could / should do the same alias thing as elsewhere to not emit the stable field names until things are actually in the spec.

Reply:
Transitional events allow to have all the same data as these types. It's just that they also provide a fallback for clients that don't support them. Of course the fallback might lose some data, but the extensible-event part will have exactly the same data.

So these events are not supposed to be used now. The MSC is pretty clear about that.

Reply:
Then we do we support them now? :P

Reply:
Because we are pioneers!

I was actually wondering if we should disable `Serialize` on these events to prevent people from sending them, but a) I'm not sure the macros will like that, b) People would need to wait for a ruma update later to send these.

Reply:
Alright if you think this is the way to go I'll trust your judgement on it. Maybe we could add a note to the docs though to not send them before the right YYYY-MM-DD?

Reply:
Yeah I was planning to add an explanation on how to use them after I'm finished implementing everything. I was wondering where to put it though. Probably on the `MessageEventContent` that is the most basic extensible type and then link to it from the other docs.

Another reason I think this type could be useful now is that we could have an abstraction layer in the SDK where people would only receive and send the final extensible event types and the SDK handles the conversion to the correct type.